### ISSUE TITLE
ui5-split-button: roledescription is not getting announced by the screen reader

### ISSUE THREAD

**Issue by unazko** [SAP/ui5-webcomponents#11893]
### Bug Description

The ui5-split-button isn't announced as a "Split button" element.

### Affected Component

ui5-split-button

### Expected Behaviour

The ui5-split-button should be announced by the screen reader as a "Split button" element.

### Isolated Example

https://stackblitz.com/edit/js-uve3cavj?file=index.html,package.json,index.js,style.css

### Steps to Reproduce

1. Open https://stackblitz.com/edit/js-uve3cavj?file=index.html,package.json,index.js,style.css
2. Start JAWS

_Observe_
Lacking type announcement for the button.

### Priority

Medium

### UI5 Web Components Version

latest

### Browser

Chrome

### Operating System

MacOS

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by unazko** [comment]
Hello @SAP/ui5-webcomponents-topic-b,

This is an issue with the `ui5-split-button`.

Best regards,
Boyan
